apps/archive/archive_test.py

```diff
@@ -477,3 +477,38 @@ def test_items_moved_to_legal_fail_if_published_queue_item_not_moved(self):
         test_items['item3'] = self.published_items[3]
         result = self.class_under_test().check_if_items_imported_to_legal_archive(test_items)
         self.assertIn('item3', result)
+
+
+class ArchiveEnhanceTestCase(TestCase):
+    def test_enhancing_archive_items(self):
+        for i in range(0, 10):
+            self.app.data.insert(ARCHIVE, [{'_id': str(i),
+                                            'type': 'text',
+                                            'linked_in_packages': [{'package': str(i + 10),
+                                                                    'package_type': 'takes'}]}])
+
+            self.app.data.insert(ARCHIVE, [{'_id': str(i + 10),
```

Comment:
What do you think about another style for formatting such things?:
```
self.app.data.insert(ARCHIVE, [{
    '_id': str(i + 10),
    'type': 'composite',
    'package_type': 'takes',
    'groups': [
        {'id': 'root', 'refs': [{'idRef': 'main'}], 'role': 'grpRole:NEP'},
        {
            'id': 'main',
            'refs': [{
                'location': ARCHIVE,
                'residRef': str(i),
                'sequence': 1,
                'type': 'text'
            }],
            'role': 'grpRole:main'
        }
    ]}
])
```
it's more readable as for me